spring-data-jpa/src/main/java/org/springframework/data/jpa/util/JpaMetamodel.java

```diff
@@ -65,12 +67,11 @@ private JpaMetamodel(Metamodel metamodel) {
 
 		this.managedTypes = Lazy.of(() -> metamodel.getManagedTypes().stream() //
 				.map(ManagedType::getJavaType) //
-				.filter(it -> it != null) //
+				.filter(Objects::nonNull) //
 				.collect(StreamUtils.toUnmodifiableSet()));
 
 		this.jpaEmbeddables = Lazy.of(() -> metamodel.getEmbeddables().stream() //
-				.map(ManagedType::getJavaType)
-				.filter(it -> it != null)
+				.map(ManagedType::getJavaType).filter(Objects::nonNull)
 				.filter(it -> AnnotatedElementUtils.isAnnotated(it, Embeddable.class))
```

Comment:
Please put a line comment at the end of the `map` operation, so that the `filter` stays on its own line.